services/cinema_hall.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models import QuerySet
+
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    cinema_halls = CinemaHall.objects.all()
+    return cinema_halls
+
+
+def create_cinema_hall(
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int
+) -> None:
+    cinema_hall = CinemaHall.objects.create(
+        name=hall_name,
+        rows=hall_rows,
+        seats_in_row=hall_seats_in_row
+    )
+    cinema_hall.save()
```

Comment:
I`m not sure, but you don`t need to call save(), because CinemaHall.objects.create should`ve already created new CInemaHall